src/main/java/io/securecodebox/persistence/defectdojo/config/Config.java

```diff
@@ -53,29 +68,39 @@ public final class Config {
     private final int maxPageCountForGets;
 
     /**
-     * Convenience constructor which sets {@link #DEFAULT_MAX_PAGE_COUNT_FOR_GETS}
+     * Convenience constructor which sets {@link #DEFAULT_REFETCH_WAIT_SECONDS} and {@link #DEFAULT_MAX_PAGE_COUNT_FOR_GETS}
      *
      * @param url    not {@code null}
      * @param apiKey not {@code null}
      */
     public Config(final @NonNull String url, final @NonNull String apiKey) {
-        this(url, apiKey, DEFAULT_MAX_PAGE_COUNT_FOR_GETS);
+        this(url, apiKey, DEFAULT_REFETCH_WAIT_SECONDS, DEFAULT_MAX_PAGE_COUNT_FOR_GETS);
     }
 
     /**
      * Dedicated constructor
      *
      * @param url                 not {@code null}
      * @param apiKey              not {@code null}
+     * @param refetchWaitSeconds  not less than 0
      * @param maxPageCountForGets not less than 1
      */
-    public Config(final @NonNull String url, final @NonNull String apiKey, final int maxPageCountForGets) {
+    public Config(final @NonNull String url, final @NonNull String apiKey, final int refetchWaitSeconds, final int maxPageCountForGets) {
```

Comment:
Is it required to have the `refetchWaitSeconds` come before the `maxPageCountForGets`?
This way this would be a breaking change.

Would be cool to have it the last parameter, and keep this constructor as is so that people who might be using this constructor would not be affected. And we can simply release this as a minor feature release version.

Reply:
That makes a lot of sense. :) 
I will change it.